test/descriptor.dart

```diff
@@ -107,6 +107,16 @@ Descriptor libPubspec(String name, String version,
   return pubspec(map);
 }
```

Comment:
I think we need some integration tests of this, showing eg. that the solver picks up pubspec_overrides.yaml and that they take precedence over dependency_overrides. They probably belong in test/version_solver_test.dart.

Reply:
Done